### ISSUE TITLE
[BITV] 9.1.1.1a/10.1 - In the "OAuth 2.0 clients" table, there is a button labeled "Show client secret." This button consistently displays the same alt text regardless of whether the "client secret" has been revealed or hidden.  (1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41824]
This should be modified. It is best to update the alt text to correspond with "Show client secret" and "Hide client secret."

`<button data-v-8f4e0666="" data-v-7be1d638="" 
aria-label="Show client secret" type="button" 
class="button-vue button-vue--icon-only button-vue--vue-tertiary-no-background">`

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-cd239865a5-v10-n1

**Comment by ShGKme** [comment]
@paramazo Please, stop spamming on every issue, especially in such a grumpy way.

All those issues are related to Nextcloud and describes the accessibility issues of Nextcloud.

[BITV Test](https://www.bitvtest.eu/) is a German project for accessibility evaluation that provides the report about accessibility issues.

We will add more information to the issues when and where it is needed. Currently, we need to create all the issues to have a general overview.
